package.json

```diff
@@ -29,9 +29,8 @@
     "preanalyze": "npm run analyze:clean",
     "prebuild": "npm run build:clean",
     "pretest": "npm run test:clean",
-    "start:prod": "cross-env BABEL_ENV=production NODE_ENV=production node server",
-    "start:tunnel": "cross-env BABEL_ENV=production NODE_ENV=production ENABLE_TUNNEL=true node server",
-    "start": "cross-env NODE_ENV=development BABEL_ENV=development node server -- --port=3001",
+    "start:prod": "npm run build && cross-env NODE_ENV=production node server",
+    "start": "cross-env NODE_ENV=development npx webpack s --config ./internals/webpack/webpack.dev.babel --port=3001 --hot --compress --history-api-fallback --watch-files=./src/**/*",
```

Comment:
Don't think `npx` is necessary here - `webpack` is already installed as a devDependency